### PR TITLE
Add support of withTotal to api-request-builder

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

The api request builder should support withTotal of the API. This is needed for performance improvements within the MC.

Reference: https://docs.commercetools.com/http-api#pagedqueryresult

### CODE REVIEW

File: docs/sdk/api/apiRequestBuilder.md [hunk 1]
```diff
@@ -95,6 +95,17 @@ const deleteUri = service.payments
   .build()
 ```
 
+#### withTotal
+
+You can also append the `withTotal` option to the uri when making a get request if you want to do not need tht total when for instance fetching orders. [More info here](https://docs.commercetools.com/release-notes#releases-2018-05-24-data-erasure)
```
Comment (emmenko) [inline docs/sdk/api/apiRequestBuilder.md]:
```
You can also append the `withTotal` option to the uri when making a query. [More info here]()
```

File: packages/api-request-builder/src/query-page.js [hunk 2]
```diff
@@ -59,3 +59,11 @@ export function perPage(value: number): Object {
   this.params.pagination.perPage = value
   return this
 }
+
+export function withTotal(value: boolean = true): Object {
```
Comment (emmenko) [inline packages/api-request-builder/src/query-page.js]: Nitpick: js doc?

File: types/sdk.js [hunk 3]
```diff
@@ -348,6 +350,7 @@ export type ServiceBuilderInstance = {
   sort: (option: string) => Object,
   page: (page: number) => Object,
   perPage: (amount: number) => Object,
+  withTotal(shouldIncludeTotal: boolean): Object,
```
Comment (emmenko) [inline types/sdk.js]: ~~`shouldIncludeTotal`~~ => `value`

**Comment by tdeekens** [comment]
Thanks the the review. I've addressed the comments and added more tests. Let me know what you think.

**Comment by tdeekens** [comment]
Thanks me darlings.

**Pull request merged**